lib/Mojo/WebSocketProxy/Dispatcher/Config.pm

```diff
@@ -0,0 +1,59 @@
+package Mojo::WebSocketProxy::Dispatcher::Config;
+
+use Mojo::Base -base;
+
+my $actions;
+my $config;
+
```

Comment:
Actually I won't like it :)
But what is wrong with it?
I'll use closure to save config.